### ISSUE TITLE
Got multiple rows in swifter.apply()

### ISSUE THREAD

**Issue by lbugnon** [jmcarpenter2/swifter#72]
Great tool! however I tried to do a simple
```python
df.Column.swifter.apply(lambda x: x**2)
```
but x is a series of Column, not a single element so the process fails. Is this the intended behavior?
**Versions:** Python 3.7.3, swifter 0.292, pandas 0.25.1

**Comment by lbugnon** [comment]
Hello Jason, sorry the late response. I tried it in two machines with the same OS/python/pandas/swifter  version and the same code. The one with Intel i7 was working as expected. However, the other one with a Ryzen 7 it is strangely returning a Series instead of an unique element. However, it is working well in a small example... I'll keep looking into it

**Comment by Chichilele** [comment]
Hi, just discovered your package while looking for parallelisers. Great work !

Similar issue: 
```
test = pd.DataFrame(np.array([[-178.1   ,   59.3   ],
       [-177.9289,  -29.2447],
       [-177.2   ,   60.4   ],
       [-175.    ,   62.1   ],
       [-174.8   ,   62.2   ]]))
test.swifter.apply(lambda x: print(x.values))
```
output
```
[[-178.1      59.3   ]
 [-177.9289  -29.2447]
 [-177.2      60.4   ]
 [-175.       62.1   ]
 [-174.8      62.2   ]]
[-178.1    -177.9289 -177.2    -175.     -174.8   ]
[ 59.3    -29.2447  60.4     62.1     62.2   ]
[-178.1]
[59.3]
[-178.1]
[59.3]
[-178.1]
[59.3]

Pandas Apply
100% 2/2 [00:00<00:00, 72.78it/s]

[-178.1    -177.9289 -177.2    -175.     -174.8   ]
[ 59.3    -29.2447  60.4     62.1     62.2   ]

0    None
1    None
dtype: object
```
Python 3.6.7 
pandas '0.23.4'
swifter '0.292'
it runs on a server, I don't know the hardware sorry!

**Comment by jakubLangr** [comment]
Hi @jmcarpenter2 , thanks for your work! 

I am experiencing a similar issue and even though I grokked that this is probably what is going on, this behavior still creates an uneven API and unexpected errors, so I would recommend giving the user an ability to control this e.g. with a kwarg.

**Comment by jmcarpenter2** [comment]
Thanks for the thoughtful response @jakubLangr. I have now fixed up PR #76, which tests that the intermediate applies have a redirected stdout. While we could add a kwarg to allow users to require non-redirected stdout during these applies, I don't really see a use case in which someone would want that. So until someone asks for that capability I'd prefer to always redirect stdout in those situations.
